sos/report/plugins/block.py

```diff
@@ -56,7 +56,14 @@ def setup(self):
             "udevadm info %(dev)s",
             "udevadm info -a %(dev)s"
         ]
-        self.add_blockdev_cmd(cmds, blacklist='ram.*')
+        self.add_blockdev_cmd(cmds, blacklist=['ram.*', 'loop.*'])
+        loop_devices = self.exec_cmd('losetup --all --noheadings')
```

Comment:
Is `--noheadings` a valid option in RHEL?

Reply:
@vikas-goel no, but this command is irrelevant now because I'm using a different approach to gather loop devices.